app/services/supplier_chooser.rb

```diff
@@ -1,14 +1,15 @@
 class SupplierChooser
-  def initialize(doorkeeper_application_owner, from_location)
-    @doorkeeper_application_owner = doorkeeper_application_owner
-    @from_location = from_location
+  attr_accessor :effective_date, :location
+
+  def initialize(effective_date, location)
+    self.effective_date = effective_date
+    self.location = location
```

Comment:
I think generally in this project we prefer using instance variables `@`, 
also, it is less ambiguous, self.effective_date could be a variable or a method, with `@effective_date` it's very clear what it is.
https://stackoverflow.com/questions/1693243/instance-variable-self-vs

Reply:
This seems to be a personal preference thing - and we use this style extensively:

<img width="672" alt="Screenshot 2020-08-25 at 13 31 10" src="https://user-images.githubusercontent.com/280472/91174521-4f3d5c80-e6d7-11ea-97bc-b0e043c8b641.png">

Reply:
👍